core/trie/trie_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/NethermindEth/juno/core/crypto"
 	"github.com/NethermindEth/juno/core/felt"
-	"github.com/NethermindEth/juno/db"
+	pebble2 "github.com/NethermindEth/juno/db/pebble"
```

Comment:
Why `pebble2` and not just `pebble` like the others?

Reply:
Because the file imports 2 packages named pebble